### PR TITLE
Fix spaces/tabulation from code on data folder

### PR DESCRIPTION
**Pull request by ramon-bernardo**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

Hey! I won't go into too any edits, just spaces.

### Changes Proposed
- Use tabs instead of 4 spaces.
- Fix some spaces (missing or lost on the code)

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Review by EPuncker — DISMISSED** [review]

**Comment by luanluciano93** [comment]
SPACES

- data\lib\core\position.lua
Linha 66: 	if  self.x >= zone.nW.x and self.x <= zone.sE.x

- data\migrations\13.lua
Linha 34: 	print("  UPDATE `houses` SET `owner` = 0 WHERE `owner` = OLD.`id`;")

- data\spells\scripts\support\food.lua
Linha 8: 	2696  -- cheese

File: data/talkactions/scripts/deathlist.lua [hunk 1]
```diff
@@ -46,7 +46,7 @@ function onSay(player, words, param)
 				if date.hour < 10 then date.hour = "0" .. date.hour end
 				if date.min < 10 then date.min = "0" .. date.min end
 				if date.sec < 10 then date.sec = "0" .. date.sec end
-				str = str .. breakline .. " " .. date.day .. getMonthDayEnding(date.day) .. " " .. getMonthString(date.month) .. " " .. date.year .. " " .. date.hour .. ":" .. date.min .. ":" .. date.sec .. "   Died at Level " .. result.getNumber(resultId, "level") .. " by " .. article .. killed_by .. "."
+				str = str .. breakline .. " " .. date.day .. getMonthDayEnding(date.day) .. " " .. getMonthString(date.month) .. " " .. date.year .. " " .. date.hour .. ":" .. date.min .. ":" .. date.sec .. " Died at Level " .. result.getNumber(resultId, "level") .. " by " .. article .. killed_by .. "."
```
Comment (ramon-bernardo) [inline data/talkactions/scripts/deathlist.lua:49]: This looks like a custom format instead of wrong but I can reverse

Comment (ranisalt) [reply data/talkactions/scripts/deathlist.lua:49]: Definitely, please revert this.

Comment (ramon-bernardo) [reply data/talkactions/scripts/deathlist.lua]: Done!

**Review by EPuncker — APPROVED** [review]

**Comment by ranisalt** [comment]
Thanks!

**Pull request merged**